roles/pulp_common/handlers/restart_all_pulp.yml

```diff
@@ -17,12 +17,22 @@
 - name: Get services
   service_facts:
 
+- name: Get services to exclude
+  set_fact:
+    services_to_exclude:
+      - 'pulpcore-worker@.service'
```

Comment:
So we exclude this to workaround a limitation in the list of services. That it includes this template service, rather than the instances of it (like @1 and @2). That would be good to put in a comment.